modal/exception.py

```diff
@@ -218,6 +218,10 @@ class ClientClosed(Error):
     pass
 
 
+class FilesystemExecutionError(Error):
```

Comment:
Just being annoying but are we consistent about whether we treat "file system" as one or two words when we camelcase? You're also using `snapshot_fs` right?

Reply:
Yep, the existing `snapshot_fs` treats it as one word.